src/marquee.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  customElement,
+  LitElement,
+  property,
+  TemplateResult,
+  css,
+  html,
+} from 'lit-element';
+import { TimelineMoment } from './interfaces';
+
+type AnimationState = 'animate' | 'stop';
+
+@customElement('mar-quee')
```

Comment:
Can we namespace this element more specific to this package? `ia-anniversary-marquee`? Custom elements get registered to the global registry so I'm trying to figure out a way we can avoid naming collisions until they release the local web component registry.

Reply:
oh ya, global naming conventions even though web components are nested in the shadow does 😅 - thanks for catching!